### DISCUSSION TITLE
Extending CppClassAnalyzerGhidraScript

### DISCUSSION THREAD

**Discussion by kevinhartman** [astrelsky/Ghidra-Cpp-Class-Analyzer#61] [Q&A] [answered]
Probably a simple enough question, but it's not clear to me what the right way is to import `CppClassAnalyzerGhidraScript` from inside Eclipse. Do I need to explicitly add a JAR dependency to have access to this type, or is there a way to resolve it some other way through the Ghidra integration?

**Comment by astrelsky** [comment, marked as answer]
I don't use Eclipse so I don't know how to do it from the IDE. However it should work if you add the jar as a runtime dependency in the gradle script similar to how the Ghidra jars are in the build gradle file in this repo.
